pkg/virt-operator/resource/generate/components/deployments.go

```diff
@@ -318,7 +319,7 @@ func NewApiServerDeployment(namespace string, repository string, imagePrefix str
 					Type:   intstr.Int,
 					IntVal: 8443,
 				},
-				Path: "/apis/subresources.kubevirt.io/" + virtv1.SubresourceGroupVersions[0].Version + "/healthz",
+				Path: path.Join("/apis/subresources.kubevirt.io/", virtv1.SubresourceGroupVersions[0].Version, "healthz"),
```

Comment:
You can drop the `/` from `"/apis/subresources.kubevirt.io/"` tail.

Reply:
Done